sdk/src/tx/txHandler.ts

```diff
@@ -37,6 +38,10 @@ const DEV_TRY_FORCE_TX_TIMEOUTS =
 
 export const COMPUTE_UNITS_DEFAULT = 200_000;
 
+const BLOCKHASH_FETCH_RETRY_COUNT = 3;
+const BLOCKHASH_FETCH_RETRY_SLEEP = 200;
+const RECENT_BLOCKHASH_STALE_TIME_MS = 2_000; // Reuse blockhashes within this timeframe during bursts of tx contruction
```

Comment:
i think there are cases where this will cause bugs from a bot trying to send a similar tx multiple times with the same blockhash

which makes me think we might not want this to be default behavior in all causes